openedx/features/course_experience/templates/course_experience/course_reviews_modules/coursetalk-reviews-fragment.html

```diff
@@ -12,6 +12,7 @@
 
 % if SHOW_REVIEWS_TOOL_FLAG.is_enabled(course.id):
     <div class="coursetalk-read-reviews">
+        <span class="fa fa-spinner fa-spin"></span>
```

Comment:
Do we need an aria attribute to hide this the way we do with most of our font awesome icons?

Reply:
good catch, adding now